src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_logging.v2beta1.json.baseline

```diff
@@ -2,7 +2,60 @@
 /**
  * BEFORE RUNNING:
  * ---------------
- * 1. If not already done, enable the Google Cloud Logging API
+ * 1. If not already done, enable the Stackdriver Logging API
+ *    and check the quota for your project at
+ *    https://console.developers.google.com/apis/api/logging
+ * 2. This sample uses Application Default Credentials for authentication.
+ *    If not already done, install the gcloud CLI from
+ *    https://cloud.google.com/sdk and run
+ *    `gcloud beta auth application-default login`
+ * 3. Install the Java client library on Maven or Gradle. Check installation
+ *    instructions at https://github.com/google/google-api-java-client.
+ *    On other build systems, you can add the jar files to your project from
+ *    https://developers.google.com/resources/api-libraries/download/logging/v2beta1/java
+ */
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.logging.v2beta1.Logging;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+
+public class LoggingExample {
+  public static void main(String args[]) throws IOException, GeneralSecurityException {
+    // Required. The resource name of the log to delete.  Example:
```

Comment:
Ideally, we would do this by detecting whether the proto doc contains an example, but that starts getting dicey if the examples are not provided in a consistent format. If we could reliably detect them, we could either delete them from the proto doc we include, or delete our own example. If we do the latter, we should try to be consistent in case parameters further down don't come with examples, and we generate inconsistent ones (I'm thinking of the discrepancy here between `projects/my-project/logs/syslog` and `billingAccounts/my-billingAccount/logs/my-log`)

I'll track this. In the meantime, unless the above is trivial to do (I expect not), having an exception for logging is fine for the purposes of this PR.